src/arch/xtensa/configs/cherrytrail_defconfig

```diff
@@ -1,4 +1,5 @@
 CONFIG_CHERRYTRAIL=y
+CONFIG_CHERRYTRAIL_DMA=n
```

Comment:
Can we make it a little bit more precise?
I would go with something like `CONFIG_CHERRYTRAIL_EXTRA_DW_DMA`.